code/game/objects/items/weapons/storage/book.dm

```diff
@@ -105,7 +105,11 @@ GLOBAL_LIST_INIT(bibleitemstates, list("bible", "koran", "scrapbook", "bible",
 		to_chat(user, "<span class='danger'>The book sizzles in your hands.</span>")
 		user.take_bodypart_damage(0,10)
 		return
-
+	
+	if (!heal_mode)
+		..()
+		return
```

Comment:
Pretty sure I need this return, though. I had an issue during testing where I would attack and heal a mob at the same time, and when I added the return it fixed that.